kythe/go/extractors/config/wrapper/gradle_build_modifier_test.go

```diff
@@ -0,0 +1,123 @@
+package wrapper
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+const testDataDir = "testdata"
+
+var multipleNewLines = regexp.MustCompile("\n{2,}")
+
+func filesEqual(got, want []byte) (bool, string) {
```

Comment:
Although it's unexported, please document this one, as it's being clever about whitespace. :)